component/src/main/java/org/wso2/carbon/event/input/adapter/mqtt/MQTTEventAdapterFactory.java

```diff
@@ -0,0 +1,156 @@
+/*
+*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+package org.wso2.carbon.event.input.adapter.mqtt;
+
+import org.wso2.carbon.event.input.adapter.core.InputEventAdapter;
+import org.wso2.carbon.event.input.adapter.core.InputEventAdapterConfiguration;
+import org.wso2.carbon.event.input.adapter.core.InputEventAdapterFactory;
+import org.wso2.carbon.event.input.adapter.core.MessageType;
+import org.wso2.carbon.event.input.adapter.core.Property;
+import org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.ResourceBundle;
+
+/**
+ * The mqtt event adapter factory class to create a mqtt input adapter
+ */
+public class MQTTEventAdapterFactory extends InputEventAdapterFactory {
+
+    private java.util.ResourceBundle resourceBundle = java.util.ResourceBundle.getBundle
+            ("org.wso2.carbon.event.input.adapter.mqtt.i18n.Resources", java.util.Locale.getDefault());
+
+    @Override
+    public String getType() {
+        return org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants.ADAPTER_TYPE_MQTT;
+    }
+
+    @Override
+    public java.util.List<String> getSupportedMessageFormats() {
+        java.util.List<String> supportInputMessageTypes = new java.util.ArrayList<String>();
+        supportInputMessageTypes.add(MessageType.TEXT);
+        supportInputMessageTypes.add(MessageType.JSON);
+        supportInputMessageTypes.add(MessageType.XML);
+
+        return supportInputMessageTypes;
+    }
+
+    @Override
+    public java.util.List<Property> getPropertyList() {
+        java.util.List<Property> propertyList = new java.util.ArrayList<org.wso2.carbon.event.input.adapter.core.Property>();
+
+        // set topic
+        Property topicProperty = new Property(
+                org.wso2.carbon.event.input.adapter.mqtt.internal.util.MQTTEventAdapterConstants.ADAPTER_MESSAGE_TOPIC);
```

Comment:
you can remove the full qualified path org.wso2.carbon.event.input.adapter.mqtt.internal.util as it's imported. You can do the change on all the places